### ISSUE TITLE
Scaling might be off

### ISSUE THREAD

**Issue by snaggen** [dmo60/CoverflowAltTab#184]
When I select a window and it zooms that window to fit the screen, it looks very strange. That animation just scales a bit then it jumps to the real size of the window. Not sure what causes this, but my guess is that I have a 4k screen with 200% scale factor. So i guess it might not take the scale factor in to consideration, causing the animation to only go half way.

**Comment by mrEDitor** [comment]
Yep, just noticed the thing too.

When you go on 100%, windows animate nicely to their positions: [x1.webm](https://user-images.githubusercontent.com/6552749/218308262-f4f7ae3a-f2a1-453a-ad39-9bc5204bdc40.webm). But if you go 200%, they animate nicely... to their 100% positions and sizes: [x2.webm](https://user-images.githubusercontent.com/6552749/218308270-2d507726-e6f7-4498-91f1-588a37299a96.webm).

**Comment by dsheeler** [comment]
I'll see if I can find time for this soon.  Thanks for reporting!

**Comment by dsheeler** [comment]
OK, sorry this took about a year. Do you guys still have this issue? I can't reproduce, so maybe I fixed it!

**Comment by dsheeler** [comment]
@mrEDitor  or @snaggen ?

**Comment by Etsuya233** [comment]
Hello, I still has this issue on the lastest Fedora and Ubuntu with the extension v70. 
My screen resolution is 3200 * 2000 and the scaling is 200%.
[screen.webm](https://github.com/dmo60/CoverflowAltTab/assets/42312804/f59d8a71-a5f6-466e-a51a-d8be9c25fed7)

**Comment by dsheeler** [comment]
@Etsuya233 are you using wayland?

**Comment by dsheeler** [comment]
I think this only occurs in wayland, not X11

**Comment by Etsuya233** [comment]
> I think this only occurs in wayland, not X11

Yes. I'm using Wayland.

**Comment by pnguyen72** [comment]
Specifically, this only happens with native Wayland apps.

**Comment by dsheeler** [comment]
Did you see in https://github.com/dsheeler/CoverflowAltTab/issues/256 that @wxmup found "fractional scaling" fixes this? 

`gsettings set org.gnome.mutter experimental-features “[‘scale-monitor-framebuffer’]”`

**Comment by pnguyen72** [comment]
Thank you for the tip, I didn't know. But turning on fractional scaling makes Xwayland apps blury, I don't think I'll take that trade-off.

**Comment by Kenzyss** [comment]
> Did you see in [#256](https://github.com/dsheeler/CoverflowAltTab/issues/256) that [@wxmup](https://github.com/wxmup) found "fractional scaling" fixes this?
> 
> `gsettings set org.gnome.mutter experimental-features “[‘scale-monitor-framebuffer’]”`

Yes, it helps, but a lot of other apps break: they become clunky and huge. So you can't call it a fix.

**Comment by wxmup** [comment]
> > Did you see in [#256](https://github.com/dsheeler/CoverflowAltTab/issues/256) that [@wxmup](https://github.com/wxmup) found "fractional scaling" fixes this?
> > `gsettings set org.gnome.mutter experimental-features “[‘scale-monitor-framebuffer’]”`
> 
> Yes, it helps, but a lot of other apps break: they become clunky and huge. So you can't call it a fix.

I guess you are talking about the electron program. You must enable wayalnd for electron through the flag to keep electron scaling normally.
